docs/home/contribute/includes.md

```diff
@@ -1,12 +1,72 @@
 ---
 approvers:
 - chenopis
-title: Tabs Example
+title: Custom Jekyll Include Snippets
 ---
 
+{% capture overview %}
+This page explains the custom Jekyll include snippets that can be used in
+Kubernetes documentation markdown.
+
+Read more about includes in the [Jekkyl documentation] (https://jekyllrb.com/docs/includes/).
+{% endcapture %}
+
```

Comment:
This link is not rendering correctly. Maybe remove the space between [ ] and ( ).